# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/stf/rte/makefile-am-behave.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./mp1e/Makefile.am.orig	2004-09-07 17:05:47.487709032 +0200
+++ ./mp1e/Makefile.am	2004-09-07 17:06:02.154479344 +0200
@@ -4,8 +4,6 @@
 INCLUDES = -D_GNU_SOURCE -D_REENTRANT -include $(top_builddir)/config.h \
 	-Wall -Wunused -Wmissing-prototypes -Wmissing-declarations
 
-if BACKEND_MP1E
-
 SUBDIRS = video audio systems common
 DIST_SUBDIRS = video audio systems common
 EXTRA_DIST =
@@ -20,28 +18,4 @@
 ## XXX remove options.h
 libmp1e_la_SOURCES = b_mp1e.c b_mp1e.h options.h
 
-else
-
-SUBDIRS = macros video audio vbi systems common devices test
-DIST_SUBDIRS = macros video audio vbi systems common devices test
-EXTRA_DIST = mp1e.lsm.in mp1e.spec.in $(man_MANS) BUGS b_mp1e.h rte
-
-dist-hook:
-	for file in b_mp1e.c; do \
-		rm -rf $(distdir)/$$file; done
-
-bin_PROGRAMS = mp1e
-
-mp1e_SOURCES = main.c options.c global_data.c options.h
-
-mp1e_LDADD = \
-	systems/libsystems.la \
-	audio/libaudio.la vbi/libvbi.la \
-	devices/libdevices.la video/libvideo.la common/libcommon.la \
-	test/libtest.la $(X_LIBS) -lm $(PTHREAD_LIB) $(ESD_LIBS)
-
-man_MANS = mp1e.1
-
-endif
-
 DISTCLEANFILES = site_def.h
